app/src/main/java/cm/aptoide/pt/analytics/AptoideNavigationTracker.java

```diff
@@ -0,0 +1,61 @@
+package cm.aptoide.pt.analytics;
+
+import cm.aptoide.pt.account.view.LoginSignUpCredentialsFragment;
+import cm.aptoide.pt.logger.Logger;
+import cm.aptoide.pt.view.search.SearchPagerTabFragment;
+import cm.aptoide.pt.view.store.GetStoreFragment;
+import cm.aptoide.pt.view.store.home.HomeFragment;
+import cm.aptoide.pt.view.wizard.WizardFragment;
+import java.util.List;
+
+/**
+ * Created by jdandrade on 06/09/2017.
+ */
+
+public class AptoideNavigationTracker implements NavigationTracker {
+
+  public static final String DO_NOT_REGISTER_VIEW = "do_not_register_view";
```

Comment:
Since NavigationTrackFragment is the one who uses the flag, it should be the one who defines the tag to be used.